### PR TITLE
revert fix, add tz aware today for air bergen

### LINKED ISSUE #162: Midnight data received from BG sensors are interpreted +24h wrong
When we are given data from BG sensors at midnight (say 00:00 the midnight between day 1 and day 2), we interpret the data to belong to next midnight (end of day 2).

### PR DESCRIPTION
**Pull request by pgdr**
fixes #162

### CODE REVIEW

**Comment by oyta** [comment]
You are right about that it's done wrong in the first place by putting the local timestamp in a UTC-string. However I don't think the solution resolves the problem. The reason is that you still put the a local timestamp in a timestring that is UTC (Zulu-timestring). You should rather set the local timestamp in a localize date time object. I have some feedback:

1. Instead of hardcoding the timezone, you should use the timezone set in Django by utilizing the Timestamp class or at least using the Django.utils.timezone by something like:
`timezone.get_current_timezone_name()`

2. You should localize the timestamp using Pytz like in the [Django documentation](https://docs.djangoproject.com/en/1.10/topics/i18n/timezones/#usage)

3. You could use `datetime.replace(hour=thehour, minute=0, second=0) ` where the hour is the timestamp scraped from luftkvalitet.info.

**Comment by pgdr** [comment]
> Instead of hardcoding the timezone, you should use the timezone set in Django

No, it has to be relative to Bergen time (luftkvalitet), which I assume is `Europe/Oslo`.

> You should localize the timestamp using Pytz like in the Django documentation

I do use `pytz`, but the timezone has to match luftkvalitet's.

> You could use datetime.replace(hour=thehour, minute=0, second=0) where thehour is the timestamp scraped from luftkvalitet.info.

I agree with that.

**Comment by oyta** [comment]
> No, it has to be relative to Bergen time (luftkvalitet), which I assume is Europe/Oslo.
> I do use pytz, but the timezone has to match luftkvalitet's.
Yes, you are right.

File: friskby/GW/air_in_bergen.py [hunk 1]
```diff
@@ -49,8 +53,6 @@ def post_data(device, df):
         for i in range(len(df)):
             if df[i][component] == komponent_id[dev_id]:
                 clock = df[i][tid]
-                if clock.strip() == '00:00':
-                    clock = '23:59'
                 ts = current_date + "T" + clock + ":00Z"
```
Comment (oyta) [inline friskby/GW/air_in_bergen.py]: If I understand this correctly, we will still get an incorrect timestamp as `clock` is the time in the local timezone but inserted in a timestring indicating Zulu time with the `Z`. Zulu time equals UTC.

**Comment by pgdr** [comment]
Hi, @oyta.  Thanks a lot for the review.  I replaced the `+z`-thing with the `datetime.replace` as you suggested.  I would like to merge it today to see if it works this midnight.

If you don't have any objections, I'll merge it within a couple of hours.

**Pull request merged**